common/config/azure-pipelines/jobs/fast-ci.yaml

```diff
@@ -26,10 +26,9 @@ pr:
 jobs:
   - template: ci-core.yaml
     parameters:
-      name: Node_16
-      nodeVersion: 16.13.0
-      buildIos: true
```

Comment:
did you mean to delete this?

Reply:
I did. I was trying to make sense of the various pipelines. Because this pipeline used the previously-latest node version, I thought I would replace it 18.x

I have a similar question open [here](https://github.com/iTwin/itwinjs-core/pull/4295/files/c5d3efb5f72a88e4fb52f3dd9e085be84594ee61#r1003401049)

Reply:
We test with the first LTS version of a major version when running the fast CI and then dedicate the latest LTS to the regression testing pipeline. Now that I type it out, seems a bit backwards so feel free to switch the versions around.

Either way we should have a one of the pipelines run the first version of Node 18 to go LTS and then one of them test the latest version of that Node 18.

Reply:
done

Reply:
I was referring specifically to the `buildIos` part

Reply:
Thanks Arun, fixed.

Reply:
resolving...